bundles/vi/insert_map.moon

```diff
@@ -33,6 +33,12 @@ insert_map = {
     state.change_mode editor, 'command'
     editor.cursor.column = math.max 1, editor.cursor.column - 1
 
+  'ctrl_[': (editor) ->
+    state.insert_edit = get_edit editor
```

Comment:
Can you please extract the code into a `exit_insert_mode` function at the top next to `get_edit`?

Reply:
I've refactored the code per your request. Let me know if there's anything else I can do!